docker-compose.yaml

```diff
@@ -55,11 +64,7 @@ services:
     environment:
       - GF_DEFAULT_APP_MODE=development
       - GF_USERS_DEFAULT_THEME=light
-      - GF_INSTALL_PLUGINS=marcusolsson-static-datasource,volkovlabs-variable-panel,volkovlabs-form-panel,volkovlabs-image-panel,grafana-llm-app
-      - GF_FEATURE_TOGGLES_ENABLE=externalServiceAccounts
-      - GF_AUTH_MANAGED_SERVICE_ACCOUNTS_ENABLED=true
-      - OPENAI_ORG_ID=
-      - OPENAI_API_KEY=
+      - GF_INSTALL_PLUGINS=marcusolsson-static-datasource
```

Comment:
don't we  need the plugins from the original docker file? and not sure about removing the others either 🤔

Reply:
oh yes, good catch, thank you! I reverted the `grafana-dev` one but missed these 🤦🏼